practice_problems_3/delete_user.php

```diff
@@ -0,0 +1,55 @@
+<?php
+session_start();
+
+if (!isset($_SESSION["id"])) {
+    header("Location: login.php");
+    exit;
+}
+
+$mysqli = require __DIR__ . "/database.php";
+$sql = "SELECT * FROM User WHERE id = ?";
+$stmt = $mysqli->prepare($sql);
+$stmt->bind_param("i", $_SESSION["id"]);
+$stmt->execute();
+$result = $stmt->get_result();
+$user = $result->fetch_assoc();
+
+// If form is submitted, delete user
+if ($_SERVER["REQUEST_METHOD"] === "POST") {
```

Comment:
This will work for every post method for this page. Add a specific method with your post request.